test/Sentry.Tests/SentrySdkTests.cs

```diff
@@ -424,9 +424,13 @@ public void ConfigureScope_Sync_CallbackNeverInvoked()
         Assert.False(invoked);
     }
 
-    [Fact]
+    [SkippableFact]
     public async Task ConfigureScope_OnTask_PropagatedToCaller()
     {
+#if __ANDROID__
+        Skip.If(true, "Flaky on Android");
```

Comment:
Android-only flaky tests too? 😬

Reply:
Yep, it's the whole mixed bag of unreliable don't really trust these device tests. To be fair, that's not just this PR/branch - our Device Tests have been unreliable pretty much ever since I joined Sentry. Initially we were running them against 6 different API levels on Android and we disabled API levels 28, 29, 30, 32 because they were so unreliable. 

I think step one is to disable the flaky tests (we just ignore those failures and rerun the tests - so they're kind of meaningless) until we can come back and make them proper tests we trust again.